README.md

```diff
@@ -16,72 +16,77 @@ You can simply install it via opam in your current switch by running:
 opam install opam-monorepo
 ```
 
-Note that once it is installed you can invoke it as an opam command:
+Note that once it's installed you can invoke it as an opam command:
 ```
 opam monorepo <subcommand> ...
 ```
 
-Note that opam knows about available plugins and will offer you to install opam-monorepo
+Note that opam knows about available plugins and will offer to install `opam-monorepo`
 if you try to invoke it without having installed it beforehand.
 
 ## Usage
 
-The basic usage for opam-monorepo is to start by running the following command from the root
+The basic usage for `opam-monorepo` is to start by running the following command from the root
 of your project:
 ```
 opam monorepo lock
 ```
 
-This will generate a project-wide lockfile under `<project-name>.opam.locked` that contains:
+This will generate a project-wide lock file under `<project-name>.opam.locked` that contains:
 - The full list of your direct and transitive opam dependencies, according to the specifications
   in the `.opam` files at your project's root, with hard version constraints (`{= <version>`) in
   the `depends` field.
-- A `pin-depends` field filled with reproducible pins, either to tarballs or to git repos with an
-  explicit commit hash for all the above dependencies, except for `ocaml`, `dune` and any virtual
+- A `pin-depends` field filled with reproducible pins, either to tarballs or to Git repos with an
+  explicit commit hash for all the above dependencies, except for `ocaml`, `dune`, and any virtual
   opam package.
 - A few extra fields meant to be interprated by `opam monorepo`
 
-This lockfile can then be consumed by the following command:
+This lock file can then be consumed by the following command:
 ```
 opam monorepo pull
 ```
 which will fetch the sources of all the locked dependencies into a `duniverse/` folder at the root of
 your project, marking them as `vendored_dirs` (see
-[dune's documentation](https://dune.readthedocs.io/en/latest/dune-files.html#vendored-dirs-since-1-11))
-so that dune will only build the artifacts you need from that folder.
+[Dune's documentation](https://dune.readthedocs.io/en/latest/dune-files.html#vendored-dirs-since-1-11))
+so that Dune will only build the artifacts you need from that folder.
 
 From that point you should be able to run `dune build` and `dune runtest` as you normally would and
 build your entire project from scratch!
 
-### opam monorepo lock
+### `opam-monorepo lock`
```

Comment:
This is not a typo and should indeed be:
```suggestion
### `opam monorepo lock`
```

Reply:
Alternatively we could rename that section to something like this:
```suggestion
### The `lock` command
```

Reply:
Whichever is most accurate! Since `opam-monorepo` is generally in monospace, I wondered if it should be `opam-monorepo` lock if not `opam-monorepo lock` ... Although I like The `lock` Command, since heading, you can also capitalise Command. ... and Lock, if it's not incorrect. Even 'The Lock Command' ?